include/vcpkg/base/cofffilereader.h

```diff
@@ -2,7 +2,7 @@
 
 #include <vcpkg/base/files.h>
 
-#include <stdint.h>
+#include <cstdint>
```

Comment:
This change is not correct: we refer to the `::global` names, not the `std::` ones. Ditto elsewhere.